server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -91,6 +94,7 @@ public TranslogTransferManager(
         this.fileTransferTracker = fileTransferTracker;
         this.logger = Loggers.getLogger(getClass(), shardId);
         this.remoteTranslogTransferTracker = remoteTranslogTransferTracker;
+        this.nodeSettings = nodeSettings;
```

Comment:
Shouldn't we be using cluster settings here? Or the node settings are the cluster settings here?

Reply:
Right, passing nodeSettings does not work, I will change it to cluster settings.

Reply:
Made the changes to add new setting to RemoteStoreSettings.